packages/dbml-parse/src/lib/interpreter/elementInterpreter/table.ts

```diff
@@ -120,8 +120,6 @@ export class TableInterpreter implements ElementInterpreter {
           return [];
         case 'indexes':
           return this.interpretIndexes(sub);
-        case 'ref':
-          return (new RefInterpreter(sub, this.env)).interpret();
```

Comment:
Should also remove the import `RefInterpreter`

Reply:
Fixed!!